File: python/hot_100/1_two-sum.py
from typing import List
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        d={}
        for i,num in enumerate(nums):
            if target-num in d.keys():
                return [d[target-num],i]
            else:
                d[num]=i

File: go/hot_100/1_two_sum.go
package main

func twoSum(nums []int, target int) []int {
	dict := map[int]int{}
	for i, v := range nums {
		if j, ok := dict[target-v]; ok {
			return []int{i, j}
		}
		dict[v] = i
	}
	return nil
}
